exchanges/gateio/gateio_websocket.go

```diff
@@ -52,20 +53,23 @@ const (
 
 	subscribeEvent   = "subscribe"
 	unsubscribeEvent = "unsubscribe"
+
+	spotOrderbookUpdateLimit = 100
 )
 
 var defaultSubscriptions = subscription.List{
 	{Enabled: true, Channel: subscription.TickerChannel, Asset: asset.Spot},
 	{Enabled: true, Channel: subscription.CandlesChannel, Asset: asset.Spot, Interval: kline.FiveMin},
-	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds},
+	// spot.order_book_update is locked to 100ms updates and 100 book size
```

Comment:
🐒  `locked` doesn't feel specific enough
```suggestion
	// spot.order_book_update only available with 100ms updates and 100 levels
```

Also: Other exchanges have limitations, e.g. update must be 1s or 100ms, levels must be 100.
I don't know if we need to single this one out for a comment. 🙂

Reply:
Yeah good point will remove